packages/dm-core-plugins/src/job/CronJob.tsx

```diff
@@ -142,10 +136,34 @@ export function ConfigureSchedule(props: {
           {interval === EInterval.HOURLY && (
             <Autocomplete
               options={[...Array(12).keys()].map((i) => i + 1)}
+              initialSelectedOptions={[Number(hourStep)]}
               label={'Hour step'}
               onInputChange={(step) => setHourStep(step)}
             />
           )}
+          <Button
+            onClick={() => setShowAdvanced(!showAdvanced)}
+            variant="ghost"
+            className={showAdvanced ? 'self-center' : 'self-end'}
```

Comment:
Since the cron-field has text underneath it, the flex centering makes the button be slightly off-center, hence the translate property
```suggestion
            className={showAdvanced ? 'self-center -translate-y-1' : 'self-end'}
```